vowpalwabbit/ccb_label.cc

```diff
@@ -161,8 +165,16 @@ void cache_label(void* v, io_buf& cache)
 void default_label(void* v)
 {
   CCB::label* ld = static_cast<CCB::label*>(v);
-  ld->outcome = nullptr;
-  ld->explicit_included_actions = v_init<uint32_t>();
+
+  // This is tested against nullptr, so unfortunately as things are this must be deleted when not used.
+  if(ld->outcome)
```

Comment:
We should probably start tagging places where better encapsulation with RAII would be nice (but is pending on figuring out how to deal with unions going forward)

Reply:
I agree